corona-warn-app/cwa-app-android
# Fix: test result keys shared fragment (EXPOSUREAPP-6694)

Added new fragment to be able to remove submitted (keys shared) tests.

How to test:
1) submit test (PCR and/or Rapid)
2) enter symptoms
3) check Home Screen
4) tap on test result card
5) check new fragment:
5.1) check correct title (PRC or Rapid)
5.2) check that for PCR there is no "Make PCR test point with icon"
